infra/bin/test-ci.ts

```diff
@@ -20,8 +20,11 @@ new TestCIStack(app, 'TestCIStack', {
 
   /* For more information, see https://docs.aws.amazon.com/cdk/latest/guide/environments.html */
   clusterConfig: {
-    botPersonalAccessToken: app.node.tryGetContext('bot_pat') || process.env.BOT_PAT,
-    webhookHMACToken: app.node.tryGetContext('webhook_hmac') || process.env.WEBHOOK_HMAC
+    githubPersonalAccessToken: app.node.tryGetContext('pat') || process.env.GITHUB_PAT,
+    githubAppId: app.node.tryGetContext('app_id') || process.env.GITHUB_APP_ID,
+    githubAppClientId: app.node.tryGetContext('client_id') || process.env.GITHUB_APP_CLIENT_ID,
+    githubAppPrivateKey: app.node.tryGetContext('app_private_key') || process.env.GITHUB_APP_PRIVATE_KEY,
+    githubAppWebhookSecret: app.node.tryGetContext('app_webhook_secret') || process.env.GITHUB_APP_WEBHOOK_SECRET
```

Comment:
nit: to keep names consistent shall we rename the context names to `github_pat`, `github_app_id` etc... ?

Reply:
@RedbackThomson had a similar comment here: https://github.com/aws-controllers-k8s/test-infra/pull/264#discussion_r1085887624  which I addressed by putting `GitHub` in the name of the class and removing all of the repetitive `github`s in variable names